salt/utils/master.py

```diff
@@ -39,6 +39,98 @@
 log = logging.getLogger(__name__)
 
 
+def get_running_jobs(opts):
+    '''
+    Return the running jobs on this minion
+    '''
+
+    ret = []
+    proc_dir = os.path.join(opts['cachedir'], 'proc')
+    if not os.path.isdir(proc_dir):
```

Comment:
Wouldn't this be considered an error? I'm wondering if we should log this at at least info level if not error?

Reply:
Perhaps a case where no jobs have run and the directory doesn't exist yet?